code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -792,6 +792,8 @@
 	processing_speed = STATUS_EFFECT_NORMAL_PROCESS
 	/// Will act as the main timer as well as changing how much damage the ants do.
 	var/ants_remaining = 0
+	/// Multiplied by the number of ants whil calculating damage
+	var/ant_damage_coeff = 0.0016
```

Comment:
should just call this damage_per_ant. 
also uh, this comment isn't all that useful but I'm not sure there's a better one